### PR TITLE
[RPS-459] Added robots.txt file to site

### PR DESCRIPTION
**Pull request by ngta1**
Enabled Robots.txt feature in essentials which installs a special
document type that can be used to read and write robots.txt.

### CODE REVIEW

**Comment by wojtek-oledzki** [comment]
from Travis:
> modified: repository-data/application/src/main/resources/hcm-content/content/documents/administration/robots.yaml
> Modified yaml files found. Run 'make format-yaml' before submitting pull request.

File: repository-data/application/src/main/resources/hcm-config/hst/configurations/default/templates.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+definitions:
+  config:
+    /hst:hst/hst:configurations/hst:default/hst:templates/robotstxt.ftl:
+      hst:script: "<#ftl encoding=\"UTF-8\" strip_whitespace=true attributes={\"content_type\"\
```
Comment (wojtek-oledzki) [inline repository-data/application/src/main/resources/hcm-config/hst/configurations/default/templates.yaml]: extract to ftl file please :)

Comment (ngta1) [reply repository-data/application/src/main/resources/hcm-config/hst/configurations/default/templates.yaml]: Extracted to separate ftl file :-)

File: cms/pom.xml [hunk 2]
```diff
@@ -170,6 +170,12 @@
             <artifactId>hippo-plugin-content-blocks</artifactId>
             <scope>compile</scope>
         </dependency>
+
+        <dependency>
+      <groupId>org.onehippo.cms7</groupId>
+      <artifactId>hippo-plugin-robotstxt-addon-repository</artifactId>
+      <scope>compile</scope>
+    </dependency>
```
Comment (jacsamell) [inline cms/pom.xml]: Alignment messed up?

File: site/pom.xml [hunk 3]
```diff
@@ -149,6 +149,12 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+      <groupId>org.onehippo.cms7</groupId>
+      <artifactId>hippo-plugin-robotstxt-hst-client</artifactId>
+      <scope>compile</scope>
+    </dependency>
```
Comment (jacsamell) [inline site/pom.xml]: Alignment?

File: cms/pom.xml (hunk 2 above)
Comment (ngta1) [reply cms/pom.xml]: good spot

**Pull request merged**